core/src/main/java/org/polypheny/db/type/entity/PolyInterval.java

```diff
@@ -27,32 +28,95 @@
 import org.jetbrains.annotations.Nullable;
 import org.polypheny.db.nodes.IntervalQualifier;
 import org.polypheny.db.type.PolySerializable;
+import org.polypheny.db.type.PolyType;
+import org.polypheny.db.type.entity.category.PolyNumber;
+import org.polypheny.db.util.temporal.TimeUnit;
 
 @EqualsAndHashCode(callSuper = true)
 @Value
 @Slf4j
+@NonFinal
 public class PolyInterval extends PolyValue {
 
 
-    public BigDecimal value;
-    public IntervalQualifier qualifier;
+    @NotNull
+    public Long millis;
+
+    @Getter
+    @NotNull
+    public Long months;
 
 
     /**
-     * Creates a PolyInterval.
+     * Creates a PolyInterval, which includes a millis and a month value, to allow for combinations liki 7-1 Month to Day, which is 7 months and 7 days.
```

Comment:
```suggestion
     * Creates a PolyInterval, which includes a millis and a month value, to allow for combinations like 7-1 Month to Day, which is 7 months and 7 days.
```
Shouldn't this be 7 month and 1 day? However, isn't the second component milis: So 7 Months and 1 millisecond?

Reply:
Fixed typo for days, added second part on how this would be represented as `PolyInterval`.